enterprise/kernel/src/main/java/org/neo4j/kernel/impl/enterprise/configuration/OnlineBackupSettings.java

```diff
@@ -40,4 +43,8 @@
 
     @Description( "Listening server for online backups" )
     public static final Setting<HostnamePort> online_backup_server = setting( "dbms.backup.address", HOSTNAME_PORT, "127.0.0.1:6362-6372" );
+
+    @Description( "Name of the SSL policy to be used by the clustering, as defined under the dbms.ssl.policy.* settings." +
+            " If no policy is configured then the communication will not be secured." )
+    public static final Setting<String> ssl_policy = prefixSetting( "dbms.backup.ssl_policy", STRING, NO_DEFAULT );
```

Comment:
We should add checks to validate this setting, so that it fails when it can't be applied (e.g. standalone enterprise). I guess we should actually add support for it to standalone enterprise in the future.